app/src/lib/stores/app-store.ts

```diff
@@ -3025,6 +3027,32 @@ export class AppStore {
     return null
   }
 
+  public _updateCurrentPullRequest(repository: Repository) {
```

Comment:
I should have caught this previously 🙈 but can this be `private` instead of `public`?

Reply:
You're right. That's an artifact from a path I abandoned.